http/server.go

```diff
@@ -541,3 +563,98 @@ func (s *Server) storeQuote(q *types.Quote) error {
 func getQuoteExpTime(q *types.Quote) time.Time {
 	return time.Unix(int64(q.AgreementTimestamp+q.TimeForDeposit), 0)
 }
+
```

Comment:
Same here, I think the name of the method should be in plural. Returns an array.